src/com/foamdev/demos/helloMVC/HelloView.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * @license
+ * Copyright 2019 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'com.foamdev.demos.helloMVC',
+  name: 'HelloView',
+  extends: 'foam.u2.Element',
```

Comment:
If it's a View, it should extend foam.u2.View, which automatically imports 'data'.